assets.src/src/redmine-tracky/controllers/form-controller.ts

```diff
@@ -9,10 +9,16 @@ export default class extends Controller {
     readonly descriptionTarget!: Element;
     readonly issueTargets!: Array<Element>;
 
+    private connected = false;
```

Comment:
I don't understand this. Why should targets be connected and the controller not? Shouldn't this work in a hierarchical way?

Reply:
First all the targets are connected -> after all targets are initially connected the 'controller' is connected.